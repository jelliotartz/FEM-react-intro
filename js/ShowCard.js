import React from 'react'

const ShowCard = React.createClass({
  render () {
    return (

    )
  }
})