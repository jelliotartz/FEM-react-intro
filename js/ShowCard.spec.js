import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ShowCard from './ShowCard'

test('ShowCard snapshot test', () => {
  const component = shallow(<ShowCard />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})


