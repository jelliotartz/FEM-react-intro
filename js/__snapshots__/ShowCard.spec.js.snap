exports[`test ShowCard snapshot test 1`] = `
<Link
  activeClassName=""
  activeOnlyWhenExact={false}
  activeStyle={Object {}}
  className=""
  isActive={[Function isActive]}
  replace={false}
  style={Object {}}
  to="/details/undefined">
  <div
    className="show-card">
    <img
      src="/public/img/posters/undefined" />
    <div>
      <h3 />
      <h4>
        (
        )
      </h4>
      <p />
    </div>
  </div>
</Link>
`;
